### PR TITLE
[IMP] delivery_ups_oca: add insurance and service package options

### PR DESCRIPTION
**Pull request by cubells**
- Add insurance option in UPS settings form.
- Add negotiated rates options in UPS settings form.
- User can change insurance value in picking form.
- Updated wrong production URL.
- Add dimensional weight compute in order to send it when no dimensional values are sent.
- Fix incorrect state code.
- Update tests
- Fix error on Canary Island country code (ISO 3166-1)
- Other minor changes

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ exclude: |
   # You don't usually want a bot to modify your legal texts
   (LICENSE.*|COPYING.*)
 default_language_version:
-  python: python3
+  python: python3.7
```
Comment (pedrobaeza) [inline .pre-commit-config.yaml:18]: Don't change this.

**Comment by pedrobaeza** [comment]
@cubells Are you still interested in this? Can you rebase and fix conflicts?

**Comment by cubells** [comment]
@pedrobaeza yes to all questions

**Comment by pedrobaeza** [comment]
Ok, waiting the rebase

**Pull request closed without merging**